packages/flame/src/hooks/useEventListener.ts

```diff
@@ -5,21 +5,18 @@ type CallbackEvent<T> = (event: T) => void;
 export function useEventListener<T extends Event>(
   eventKey: string,
   callback: CallbackEvent<T>,
-  target?: EventTarget,
+  target: EventTarget = document,
 ) {
   React.useEffect(() => {
     const captureEvent: CallbackEvent<T> = event => {
       callback(event);
     };
 
-    if (target || document) {
-      (target || document).addEventListener(eventKey, captureEvent as any);
+    if (target) {
+      target.addEventListener(eventKey, captureEvent as any);
+      return () => {
+        target.removeEventListener(eventKey, captureEvent as any);
+      };
     }
-
-    return () => {
-      if (target || document) {
-        (target || document).removeEventListener(eventKey, captureEvent as any);
-      }
-    };
   }, [callback]);
```

Comment:
Hello Tim! Good comment! I'll propose a change in this direction. For updating the callbackRef, I think the recommend way would be`useEffect`. But to be honest, I can't think of an actual use case that could cause an issue if you update it in the body... 🤷🏻‍♂️

Reply:
Hey! 👋 the use cases i has in mind were more when the `target` could be possibly smaller elements like third party library elements or potentially the `<main>` of a page instead of a body etc. but maybe I totally over thought this and document is all this would be use for haha!

Reply:
I did not fully get your last comment @Multimo 🤔